policyengine_us/variables/gov/states/az/tax/income/credits/property_tax_credit/az_property_tax_credit_eligible.py

```diff
@@ -0,0 +1,29 @@
+from policyengine_us.model_api import *
+
+
+class az_property_tax_credit_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Eligible for the Arizona Property Tax Credit"
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(period).gov.states.az.tax.income.credits.property_tax
+
+        age = person("age", period)
+        age_eligible = age >= p.min_age
+
+        ssi = person("ssi", period)
+        ssi_qualifies = ssi > 0
+
+        property_tax = person("real_estate_taxes", period)
+        rent = person("rent", period)
+        payment_qualifies = (property_tax + rent) > 0
+
+        head = person("is_tax_unit_head", period)
```

Comment:
The tax unit can't qualify via the spouse?

Reply:
We have interpreted line 8 as that only the head can qualify

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/49603b16-77b0-45f8-bf6d-88def18bf882)

Reply:
Thanks. I think we should still use head or spouse age. Currently we assign head within tax units arbitrarily, but this would create an incentive to have the 65+ year old file as head. Using both head and spouse age would thus better model reality. The only accuracy compromise is that the senior has to be the one to themselves pay rent or property tax, but if they're married I would expect that they'd generally pay these together, so I think it's worth it.